### PR TITLE
Update V.4.6.0

### PR DESCRIPTION
**Pull request by bagelwastaken**
# Changelog

- Fixed all of the known errors that have been caused by the conflict resolve in https://github.com/bagelwastaken/kekbot/pull/24#pullrequestreview-1295942949

- Added Chatbot feature.

Commits: https://github.com/bagelwastaken/kekbot/commit/2eea356e9d51f1b43107241e201d92e216a87590 , https://github.com/bagelwastaken/kekbot/commit/dca856428b68e0fd647f3709aaa0bb7fadcf0258

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,8 @@
 	},
 	"homepage": "https://github.com/bagelwastaken/kekbot#readme",
 	"dependencies": {
+		"@discordjs/builders": "^1.4.0",
```
Comment (ThatsLiamS) [inline package.json:20]: Why is DiscordJS/Builders being used in conjunction with **v14 of Discord.js**

File: src/commands/chat/chat.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
```
Comment (ThatsLiamS) [inline src/commands/chat/chat.js:1]: use the builder from the DiscordJS package
```js
const { SlashCommandBuilder } = require('discord.js')
```

File: src/commands/chat/chat.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+			await axios.get(`http://api.brainshop.ai/get?bid=${bid}&key=${key}&uid=${user_id}&msg=${user_input}`)
+				.then(res => {
+					const data = res.data;
+					const reply = data.cnt;
+
+					if (reply) {
+						interaction.followUp(reply);
+					}
+					else if (!reply) {
+						interaction.followUp('Hmmm, I don\'t know how to respond to that...');
```
Comment (ThatsLiamS) [inline src/commands/chat/chat.js:42]: Send the MessagePayload shown here <https://discord.js.org/#/docs/discord.js/main/class/MessagePayload>

```js
interaction.followUp({ content: 'Hmmm, I don\'t know how to respond to that...' });
```
Same with line 39

**Review by ThatsLiamS — CHANGES_REQUESTED** [review]

File: src/commands/chat/chat.js (hunk 2 above)
Comment (bagelwastaken) [reply src/commands/chat/chat.js:1]: Understood 👍

File: package.json (hunk 1 above)
Comment (ThatsLiamS) [reply package.json:20]: Please ensure that any unused dependencies are removed from the project

**Review by ThatsLiamS — CHANGES_REQUESTED** [review]

**Review by ThatsLiamS — APPROVED** [review]
Ready to go :)

**Pull request merged**